### PR TITLE
Vula 3368

### PR DESCRIPTION
**Pull request by axxter99**
search inactiveStaff & inactive

### CODE REVIEW

File: tool/src/java/org/sakaiproject/profilewow/tool/producers/SearchResultProducer.java [hunk 1]
```diff
@@ -308,6 +309,34 @@ else if ((profile.getHidePublicInfo() != null) && (profile.getHidePublicInfo().b
...
+		  String TYPE_STAFF = "staff";
+		  String TYPE_THIRDPARTY = "thirdparty";
+		  //String TYPE_OFFER = "offer";
+		  
+		log.debug("User: " + profile + "Type: " + profile.getTypeUuid());
+		User user = null;
+		try {
+			user = userDirectoryService.getUser(profile.getAgentUuid());
+			String type = user.getType();
+			log.debug("User: " + user.getDisplayId() + ", type: "+ user.getType());
```
Comment (smarquard) [inline tool/src/java/org/sakaiproject/profilewow/tool/producers/SearchResultProducer.java]: I think nowadays this works with placeholders, e.g.

			log.debug("User: {}, type: {}", profile, user.getType());

File: tool/src/java/org/sakaiproject/profilewow/tool/producers/SearchResultProducer.java [hunk 2]
```diff
@@ -308,6 +309,34 @@ else if ((profile.getHidePublicInfo() != null) && (profile.getHidePublicInfo().b
 		return searchResults;
 	}
 
+	private boolean getSakai(SakaiPerson profile) {
```
[the remark this answers is not in the document]
Comment (axxter99) [reply tool/src/java/org/sakaiproject/profilewow/tool/producers/SearchResultProducer.java]: OK
String TYPE_OFFER = "offer":?

[the remark this answers is not in the document]
Comment (smarquard) [reply tool/src/java/org/sakaiproject/profilewow/tool/producers/SearchResultProducer.java]: I think we should leave out "offer", so leave it as-is (commented-out).

**Comment by smarquard** [comment]
Looks good, thanks!

**Pull request merged**